src/components/ec-checkbox/ec-checkbox.vue

```diff
@@ -0,0 +1,212 @@
+<template>
```

Comment:
unchecked :focus is not working. If I tab into unchecked checkbox I don't see any visual change.